02-oop-inheritance/javaScript/downloableMovie.js

```diff
@@ -0,0 +1,59 @@
+function MovieObserver() {
```

Comment:
all functions definitions must be defined into a variable, also if it is stored into a variable you should use a anonymous function